src/main/scala/temple/generate/kube/KubernetesGenerator.scala

```diff
@@ -39,8 +40,46 @@ object KubernetesGenerator {
   private def generateDbService(service: Service): String =
     generateHeader(service, GenType.Service, isDb = true)
 
-  private def generateDbDeployment(service: Service): String =
-    generateHeader(service, GenType.Deployment, isDb = true)
+  private def generateDbDeployment(service: Service): String = {
+    val name = service.name + "-db"
+    val deploymentBody = Body(
+      DeploymentSpec(
+        1,
+        Selector(Labels(service.name, GenType.Deployment, isDb = true)),
+        strategy = Some(Strategy("Recreate")),
```

Comment:
Wondering if `Strategy` could be predefined? And we just use `Strategy.Recreate`?